bindings/core/src/lib.rs

```diff
@@ -44,7 +43,6 @@ pub fn init_logger(config: String) -> std::result::Result<(), fern_logger::Error
 #[derivative(Debug)]
 #[serde(rename_all = "camelCase")]
 pub struct WalletOptions {
-    pub address: Option<Bech32Address>,
```

Comment:
This should stay, users should be able to provide the address or the bip_path

Reply:
Are we bringing tis back then?

Reply:
not sure why I removed it in the first place :thinking:

Reply:
reverted